src/encoded/types/treatment.py

```diff
@@ -99,6 +99,8 @@ def display_title(self, request, treatment_type=None, chemical=None,
 
         if chemical:
             dis_tit = chemical + " treatment" + conditions
+        elif biological_agent:
+            dis_tit = biological_agent + " infection" + conditions
```

Comment:
My only concern here is that not all treatment with biological agents may be infections - even though it's a little awkward might be more future proof to just keep it as  Virus treatment?